examples/fluids/index.md

```diff
@@ -634,19 +634,26 @@ The `STGRand.dat` file is the table of the random number set, $\{\bm{\sigma}^n,
 The following table is presented to help clarify the dimensionality of the
 numerous terms in the STG formulation.
 
-| Math            | Label  | $f(\bm{x})$? | $f(n)$? |
-|-----------------|--------|--------------|---------|
-| $ \{\bm{\sigma}^n, \bm{d}^n, \phi^n\}_{n=1}^N$        | RN Set | No           | Yes     |
-| $\bm{\overline{u}}$ | ubar | Yes | No |
-| $U_0$           | U0     | No           | No      |
-| $l_t$           | l_t    | Yes          | No   |
-| $\varepsilon$   | eps    | Yes          | No   |
-| $\bm{R}$        | R_ij   | Yes          | No      |
-| $\bm{C}$        | C_ij   | Yes          | No      |
-| $q^n$           | q^n    | Yes           | Yes     |
-| $\{\kappa^n\}_{n=1}^N$ | k^n  | No           | Yes      |
-| $h_i$           | h_i    | Yes          | No   |
-| $d_w$           | d_w    | Yes          | No   |
+| Math                                           | Label    | $f(\bm{x})$?   | $f(n)$?   |
+| -----------------                              | -------- | -------------- | --------- |
+| $ \{\bm{\sigma}^n, \bm{d}^n, \phi^n\}_{n=1}^N$ | RN Set   | No             | Yes       |
+| $\bm{\overline{u}}$                            | ubar     | Yes            | No        |
+| $U_0$                                          | U0       | No             | No        |
+| $l_t$                                          | l_t      | Yes            | No        |
+| $\varepsilon$                                  | eps      | Yes            | No        |
+| $\bm{R}$                                       | R_ij     | Yes            | No        |
+| $\bm{C}$                                       | C_ij     | Yes            | No        |
+| $q^n$                                          | q^n      | Yes            | Yes       |
+| $\{\kappa^n\}_{n=1}^N$                         | k^n      | No             | Yes       |
+| $h_i$                                          | h_i      | Yes            | No        |
+| $d_w$                                          | d_w      | Yes            | No        |
+
+#### Internal Damping Layer (IDL)
+STG creates spurious acoustic waves during it's development to natural
+turbulent structures. To damp those out, we use an internal damping layer (IDL)
+to prevent them from growing. This implementation was inspired from
+{cite}`shurSTG2014`, but is implemented here as a ramped volumetric forcing
+term in the residual itself.
```

Comment:
I would say more here, but I don't fully grasp the analytical form of the damping (particularly the use of `StateFromY_fwd` instead of just `StateFromY` for `damp_s`).

Reply:
Similar to below, we use
```math
-\sigma(\symbf x) \underbrace{A_0 Y'}_{U'}
```
where $Y' = \\{p - p_{ref}, \symbf 0, 0\\}$.
![image](https://user-images.githubusercontent.com/3303/224119322-fae86ec8-913c-46b3-9a95-bb777e9ca8ae.png)

Reply:
Should we perhaps denote $\symbf{Y}'$ as $\mathrm{d}\symbf{Y}$ or $\Delta \symbf{Y}$?

Also, I don't think we're exactly doing $\symbf{A}_0 \symbf{Y}$ either, as that would be equivalent to `StateFromY(gas, damp_y, x_i)`. Instead we're doing `StateFromY_fwd(context, s, damp_Y, x_i, dx_i)`, which I don't believe has a representation in our normal notation. Since we're only changing pressure, the two operations (`StateFromY` and `StateFromY_fwd`) are equivalent, but I think the intention here is important (and I'm not sure what the intention is).

Reply:
$A_0$ is $\partial U/\partial Y$ linearized at the current state so I think that's what we're doing. I felt like it was simpler (closer to linear), but probably either would work. I don't have a sufficiently discerning test case.

Reply:
My assertion that we're not doing $A_0 \symbf{Y}$ comes from the fact that $A_0$ is used in the literature for $Y_{,t}$, but I see now that we do use `StateFromQi_fwd` for the $Y_{,t}$ term. So I think you're right that $A_0 Y$ is equivalent to `StateFromQi_fwd`.

Reply:
Ok, I think I see my confusion now. `StateFromQi` computes $U(Y)$, while `StateFromQi_fwd` computes $\partial U(Y) / \partial Y$. 

For completeness sake, when I'm talking about `StateFrom*`, I'm really talking about `StateConservativeFromPrimitive*`, which is used inside `StateFromY*`.

Reply:
Will you add a couple sentences here?

Reply:
Yeah, I started on that, but didn't finish it up.